app/src/main/java/erevacation/com/userlist/ui/homescreen/profile/ProfileFragment.kt

```diff
@@ -8,12 +8,21 @@ import erevacation.com.userlist.databinding.FragmentProfileBinding
 class ProfileFragment : BasicFragment<ProfileContract.ProfilePresenter, FragmentProfileBinding>()
         ,ProfileContract.ProfileView {
 
+    var user: String = ""
+
     override fun getLayoutId(): Int = R.layout.fragment_profile
 
+    override fun onStart() {
+        this.user = arguments!!.getString(USER)
+        super.onStart()
+    }
+
     companion object {
+        private val USER = "user"
         const val PROFILE_FRAGMENT_TAG: String = "THEME_FRAGMENT_TAG"
```

Comment:
This string is not accurate. It should be probably `const val PROFILE_FRAGMENT_TAG: String = "PROFILE_FRAGMENT_TAG"`